jscomp/bsb/bsb_ninja_rule.ml

```diff
@@ -139,6 +138,8 @@ let make_custom_rules
         Buffer.add_string buf " ";
         Buffer.add_string buf dev_incls;
     end;
+    if error_syntax_kind = Some Bsb_db.Reason then
+      Buffer.add_string buf " -bs-re-out";
```

Comment:
Technically `-bs-re-out` could be doing more things, but we know it doesn't right know, maybe the variable name could be `impl_syntax_kind` because of it.